.github/workflows/release-please.yml

```diff
@@ -0,0 +1,23 @@
+on:
```

Comment:
do we still need a workflow file? I thought the bot was handling that at this point.

Reply:
Wow. I didn't know about this. I'll go ahead and delete it